views/layout.pug

```diff
@@ -1,27 +1,27 @@
 doctype html
 html(lang='en')
 
-  head
-    meta(charset='utf-8')
-    meta(http-equiv='X-UA-Compatible' content='IE=edge')
-    meta(name='viewport' content='width=device-width, initial-scale=1')
+    head
+        meta(charset='utf-8')
+        meta(http-equiv='X-UA-Compatible' content='IE=edge')
+        meta(name='viewport' content='width=device-width, initial-scale=1')
 
-    link(rel='stylesheet' href='bootstrap/css/bootstrap.min.css')
-    link(rel='stylesheet' href='materializecss/css/materialize.min.css')
-    link(rel='stylesheet' href='animate/animate.min.css')
-    link(rel='stylesheet' href='/css/icons.css')
-    link(rel='stylesheet' href='/css/common.css')
-    link(rel='stylesheet' href='/css/navbar.css')
+        link(rel='stylesheet' href='bootstrap/css/bootstrap.min.css')
+        link(rel='stylesheet' href='materializecss/css/materialize.min.css')
+        link(rel='stylesheet' href='animate/animate.min.css')
+        link(rel='stylesheet' href='/stylesheets/css/icons.css')
+        link(rel='stylesheet' href='/stylesheets/css/common.css')
+        link(rel='stylesheet' href='/stylesheets/css/navbar.css')
 
-    script(src='jquery/jquery.js')
-    script(src='bootstrap/js/bootstrap.min.js')
-    script(src='materializecss/js/materialize.min.js')
-    script(src='/js/common.js')
+        script(src='jquery/jquery.js')
+        script(src='bootstrap/js/bootstrap.min.js')
+        script(src='materializecss/js/materialize.min.js')
+        script(src='/js/common.js')
 
-    block styles
-    block title
-      title Velocity
+        block styles
+        block title
+            title Velocity
 
-  body
-    block content
-    block scripts
\ No newline at end of file
+    body.theme-default
```

Comment:
we need this as user.theme, but we can leave this for another PR

Reply:
i got uu